spec/core_functions/color/hsla.hrx

```diff
@@ -56,3 +50,89 @@ a {
 a {
   b: rgba(51, 204, 204, 0.4);
 }
+
+<===>
+================================================================================
+<===> with_css_var/first_arg/input.scss
+a {b: hsla(var(--a), 25%, 12%, 0.4)}
```

Comment:
We already test this for `hsl()`, so we don't need to test it for `hsla()` as well. See https://github.com/sass/sass-spec/blob/4016da98530de32c2783545a2fc61e23ba70f4f7/spec/core_functions/color/hsla.hrx#L1-L3.

Reply:
gotcha, I totally missed the `hsl/` directory